gatk_base_recalibrator_4.1.8.1/gatk_base_recalibrator_4.1.8.1.cwl

```diff
@@ -0,0 +1,349 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: gatk_base_recalibrator_4_1_8_1
+baseCommand:
+  - gatk
+inputs:
+  - id: input
+    type: File
+    inputBinding:
+      position: 3
+      prefix: '--input'
+    doc: BAM/SAM file containing reads
+    secondaryFiles:
+      - ^.bai
+  - id: known_sites_1
+    type: File
+    inputBinding:
+      position: 3
+      prefix: '--known-sites'
+    doc: >-
+      One or more databases of known polymorphic sites used to exclude regions
+      around known polymorphisms from analysis
+    secondaryFiles:
+      - .idx
+  - id: reference
+    type: File
+    inputBinding:
+      position: 3
+      prefix: '--reference'
+    doc: Reference sequence file
+    secondaryFiles:
+      - .fai
+      - ^.dict
+  - id: output_file_name
+    type: string?
+    doc: Output file name. Not Required
+  - id: add_output_sam_program_record
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--add-output-sam-program-record'
+  - id: add_output_vcf_command_line
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--add-output-vcf-command-line'
+  - id: arguments_file
+    type:
+      - 'null'
+      - type: array
+        items: File
+        inputBinding:
+          position: 0
+          prefix: '--arguments_file'
+  - id: binary_tag_name
+    type: string?
+    inputBinding:
+      position: 10
+      prefix: '--binary-tag-name'
+  - id: bqsr_baq_gap_open_penalty
+    type: float?
+    inputBinding:
+      position: 10
+      prefix: '--bqsr-baq-gap-open-penalty'
+  - id: cloud-index-prefetch-buffer
+    type: int?
+    inputBinding:
+      position: 10
+      prefix: '--cloud-index-prefetch-buffer'
+  - id: cloud_prefetch_buffer
+    type: int?
+    inputBinding:
+      position: 10
+      prefix: '--cloud-prefetch-buffer'
+  - id: create_output_bam_index
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--create-output-bam-index'
+  - id: create_output_bam_md5
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--create-output-bam-md5'
+  - id: create_output_variant_index
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--create-output-variant-index'
+  - id: create_output_variant_md5
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--create-output-variant-md5'
+  - id: default_base_qualities
+    type: int?
+    inputBinding:
+      position: 10
+      prefix: '--default-base-qualities'
+  - id: deletions_default_quality
+    type: int?
+    inputBinding:
+      position: 10
+      prefix: '--deletions-default-quality'
+  - id: disable_bam_index_caching
+    type: boolean?
+    inputBinding:
+      position: 10
+      prefix: '--disable-bam-index-caching'
+  - id: disable_read_filter
```

Comment:
I can't tell from the gatk documentation if this option should be specified more than once for each read filter you want disabled, or if all the filters should be supplied at once as space delimited or something.

But i see below you set the `read_filter` option to be an array type where `--read-filter` is specified once for each read filter you want (e.g. --read-filter read_filter1 --read-filter read_filter2, etc.)

However, `disable_read_filter` and `read_filter` seem to have the same input type in the gatk documentation, but you seem to switch back and forth between array and string. so I am not sure which is correct.